Quickstart.md

```diff
@@ -0,0 +1,197 @@
+# QuickStart guide on how to setup AppCli with a sample application
```

Comment:
This PR is missing a `CHANGELOG.md` entry. Could you please add a new entry to the `Added` block for unreleased?